contrib/python/podman/podman/libs/images.py

```diff
@@ -40,53 +40,53 @@ def create(self, *args, **kwargs):
         """
         details = self.inspect()
 
-        config = Config(image_id=self.id, **kwargs)
+        config = Config(image_id=self._id, **kwargs)
         config['command'] = details.containerconfig['cmd']
         config['env'] = self._split_token(details.containerconfig['env'])
         config['image'] = copy.deepcopy(details.repotags[0])
         config['labels'] = copy.deepcopy(details.labels)
         config['net_mode'] = 'bridge'
         config['network'] = 'bridge'
 
-        logging.debug('Image {}: create config: {}'.format(self.id, config))
+        logging.debug('Image %s: create config: %s', self._id, config)
         with self._client() as podman:
-            id = podman.CreateContainer(config)['container']
-            cntr = podman.GetContainer(id)
-        return Container(self._client, id, cntr['container'])
+            id_ = podman.CreateContainer(config)['container']
```

Comment:
I'm not a big fan of a dash at the end of a bare variable like this.  Could we at least change this to "ctrid" or some such.  Ditto inputt_ below, perhaps "inputStream".  Granted nothing wrong with it as is, the comment can be ignored, just a sweeney-twitch i thought I'd throw out for your consideration.

Reply:
@TomSweeneyRedHat ack, I followed the PEP8 recommendation and I'm not sure I like it.